app/src/main/java/com/codesoom/assignment/application/ProductService.java

```diff
@@ -12,43 +13,65 @@
 @Service
 @Transactional
 public class ProductService {
+    private final Mapper mapper;
     private final ProductRepository productRepository;
 
-    public ProductService(ProductRepository productRepository) {
+    public ProductService(Mapper dozerMapper , ProductRepository productRepository) {
+        this.mapper = dozerMapper;
         this.productRepository = productRepository;
     }
 
+    /**
+     * 모든 상품들을 반환한다
+     * @return List<Product>
+     */
     public List<Product> getProducts() {
         return productRepository.findAll();
     }
 
+    /**
+     * 식별자에 해당하는 상품을 반환한다
+     * 식별자에 해당하는 상품이 없다면 상품을 찾을 수 없다는 예외를 던진다
+     * @param id
+     * @return Product
+     * @throws ProductNotFoundException
+     */
```

Comment:
이렇게 해 보세요.

```suggestion
    /**
     * 상품을 찾아 반환한다.
     *
     * @param id 찾으려는 상품의 식별자
     * @return Product 찾은 상품
     * @throws ProductNotFoundException 식별자에 해당하는 상품이 없는 경우
     */
```

JavaDoc을 가장 확실히 배우는 방법은 Java의 빌트인 클래스들을 열어보는 겁니다. 가장 먼저 읽어볼 것으로 추천하는 것은 `String.java`에요. 꼭 읽어보세요.